### PR TITLE
refactor(android): update CSSNode references → shadowNode

### PR DESCRIPTION
**Pull request by swashcap**
## Motivation

I‘ve been digging through the _ReactAndroid_ implementation to make some great react-native ↔ android native components. I’ve come across references to `CSSNode`, which appears to have been replaced in b9cedaefa61bf1840ef8a7c5e621da4789df8c80. I _think_ shadow nodes replaced these `CSSNodes`, so docblocks and method argument names are updated in this PR to avoid confusion.

## Test Plan

This only changes argument names and docblocks. If CI passes we should be :+1: :+1:

## Release Notes

[ANDROID][ENHANCMENT][uimananger,flat]: Update old `CSSNode` references

### CODE REVIEW

File: ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactStylesDiffMap.java [hunk 1]
```diff
@@ -22,11 +22,12 @@
  * some of the accessor methods of {@link ReadableMap} by adding a default value property
  * such that caller is enforced to provide a default value for a style property.
  *
- * Instances of this class are used to update {@link View} or {@link CSSNode} style properties.
- * Since properties are generated by React framework based on what has been updated each value
- * in this map should either be interpreted as a new value set for a style property or as a "reset
- * this property to default" command in case when value is null (this is a way React communicates
- * change in which the style key that was previously present in a map has been removed).
+ * Instances of this class are used to update {@link View} or {@link ReactShadowNode} style
```
Comment (swashcap) [inline ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactStylesDiffMap.java]: Reflow to make this <=100 chars per line. The change is `s/CSSNode/ReactShadowNode`.

File: ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java [hunk 2]
```diff
@@ -60,12 +60,12 @@
  * hierarchy through {@link NativeViewHierarchyManager} at the end of each transaction.
  *
  * <p>
- * <h2>== CSSNodes ==</h2>
+ * <h2>== Shadow Nodes ==</h2>
  * In order to allow layout and measurement to occur on a non-UI thread, this module also
- * operates on intermediate CSSNodeDEPRECATED objects that correspond to a native view. These CSSNodeDEPRECATED are able
```
Comment (swashcap) [inline ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java]: `CSSNodeDEPRECATED` is how I noticed the problem!

**Comment by swashcap** [comment]
I see the following when trying to build the Android test container via `npm run test-android-build`:

```
Step 10/31 : RUN buck fetch ReactAndroid/src/test/java/com/facebook/react/modules
 ---> Running in dcdc1c82a17c
Using watchman.
Parse error for build file /app/ReactAndroid/src/test/java/com/facebook/react/modules/BUCK:
KeyError: 'CIRCLECI'
Call stack:
  File "/opt/buck/build/classes/buck_parser/buck.py", line 1213, in process_with_diagnostics
    diagnostics=diagnostics)
  File "/opt/buck/build/classes/buck_parser/buck.py", line 1103, in process
    implicit_includes=self._implicit_includes)
  File "/opt/buck/build/classes/buck_parser/buck.py", line 1036, in _process_build_file
    implicit_includes=implicit_includes)
  File "/opt/buck/build/classes/buck_parser/buck.py", line 958, in _process
    exec(code, module.__dict__)
  File "/app/ReactAndroid/src/test/java/com/facebook/react/modules/BUCK", line 41
    react_native_tests_target("java/com/facebook/react/bridge:testhelpers"),
  File "/app/ReactAndroid/DEFS", line 89, in rn_robolectric_test
    if os.path.isdir("/dev/shm") and not os.environ['CIRCLECI']:
  File "/opt/buck/build/classes/buck_parser/buck.py", line 571, in wrapper
    return real(varname, *arg, **kwargs)
  File "/usr/lib/python2.7/UserDict.py", line 40, in __getitem__
    raise KeyError(key)

The command '/bin/sh -c buck fetch ReactAndroid/src/test/java/com/facebook/react/modules' returned a non-zero code: 1
```

Any thoughts? Looks like Circle threw on something else.

**Comment by swashcap** [comment]
Cool, I'll look into rebasing.

**Comment by cpojer** [comment]
Thanks for this PR. It seems like unfortunately we never got time to look at this to merge it promptly and now it is quite outdated so I'll be closing it.

**Pull request closed without merging**